tests/cases/compiler/duplicateErrorNameNotFound.ts

```diff
@@ -0,0 +1,7 @@
+// @strict: true
+
+type RoomInterfae = {};
+
+export type {
+    RoomInterface
+}
```

Comment:
Did these tests actually produce duplicates, or were they just useful in getting near the code path for debugging?

Reply:
They produced duplicates, yes.